TestSummaryFiles.cmake

```diff
@@ -0,0 +1,44 @@
+# Set absolute tolerance to be used for testing
+set(tol 1e-2)
+
+# Define some paths
+set(RESULT_PATH ${PROJECT_BINARY_DIR}/tests/results)
```

Comment:
I don't know how well it fits in with this model - but I have used the [TestArea](https://github.com/Ensembles/ert/blob/master/devel/libert_utilxx/src/TestArea.cpp) quite a lot - and find it very convenient. The `TestArea`class works like this:

1. Create a random temporary directory and do a `chdir( )` there.
2. Optionally you can use utility functions which are aware of your current cwd and the original cwd to conveniently copy files and directories.
3. When it goes out of scope the directory and the files are removed[1,2]

Example usage:
```C++
#include <ert/util/TestArea.hpp>
...
{
    ERT::TestArea ta("my_funny_test");
    // Now cwd is in some random directory().
    ta.copyFile( "path/relative/to/original/cwd");
    // Do your stuff ..
}
// Now the original CWD has been recovered, and the temporary directory has been wiped.
```


[1]:  There will be a directory stem from `/tmp/$USER/...` left behind.
[2]:  For debugging purposes you can set the `store`flag to `true` - in which case the directory will be left intact.